test/e2e/specs/editor/various/rtl.spec.js

```diff
@@ -77,51 +98,75 @@ describe( 'RTL', () => {
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.press( 'Delete' );
 
-		expect( await getEditedPostContent() ).toMatchSnapshot();
+		//Check the content
+		const content = await editor.getEditedPostContent();
+		expect( content ).toBe(
+			`<!-- wp:paragraph -->
+<p>٠١</p>
+<!-- /wp:paragraph -->`
+		);
 	} );
 
-	it( 'should arrow navigate between blocks', async () => {
+	test( 'should arrow navigate between blocks', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
 		await page.keyboard.press( 'Enter' );
 
 		await page.keyboard.type( ARABIC_ZERO );
 		await page.keyboard.press( 'Enter' );
 		await page.keyboard.type( ARABIC_ONE );
-		await pressKeyWithModifier( 'shift', 'Enter' );
+		await pageUtils.pressKeyWithModifier( 'shift', 'Enter' );
 		await page.keyboard.type( ARABIC_TWO );
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.press( 'ArrowRight' );
 
 		// Move to the previous block with two lines in the current block.
 		await page.keyboard.press( 'ArrowRight' );
-		await pressKeyWithModifier( 'shift', 'Enter' );
+		await pageUtils.pressKeyWithModifier( 'shift', 'Enter' );
 		await page.keyboard.type( ARABIC_ONE );
 
 		// Move to the next block with two lines in the current block.
 		await page.keyboard.press( 'ArrowLeft' );
 		await page.keyboard.type( ARABIC_ZERO );
-		await pressKeyWithModifier( 'shift', 'Enter' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
+		await pageUtils.pressKeyWithModifier( 'shift', 'Enter' );
+
+		//Check the content
+		const content = await editor.getEditedPostContent();
+		expect( content ).toBe(
+			`<!-- wp:paragraph -->
+<p>٠<br>١</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>٠<br>١<br>٢</p>
+<!-- /wp:paragraph -->`
+		);
 	} );
 
-	it( 'should navigate inline boundaries', async () => {
+	test( 'should navigate inline boundaries', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
 		await page.keyboard.press( 'Enter' );
 
 		// Wait for rich text editor to load.
 		await page.waitForSelector( '.block-editor-rich-text__editable' );
```

Comment:
Could we change this selector to something accessible? For instance, selecting it using a role selector, or replacing it along with the `press( 'Enter' )` above with `page.click( 'role=button[name="Add default block"i]' )`.

Reply:
We still need to refactor this line to use role selectors instead though. Or maybe just remove this line if it's not doing anything meaningful now?

Reply:
Hi @kevin940726, We have refactored that line and Added the role selector https://github.com/WordPress/gutenberg/blob/09e40c9fd269154b61ad1bf8ee850ce4719bf987/test/e2e/specs/editor/various/rtl.spec.js#L154. We can't remove that line I guess.

Reply:
I mean to remove this line.

```js
await page.waitForSelector( '.block-editor-rich-text__editable' );
```

I don't think we need it anymore?